### PR TITLE
meetings: add minutes from 2023-07-15

### PR DESCRIPTION
**Pull request by attie**

### CODE REVIEW

**Review by whitequark — APPROVED** [review]

**Comment by attie** [comment]
I'd like to add previous / outstanding issues, please wait for merge.

**Comment by attie** [comment]
@whitequark - thanks, good to go.

File: docs/meetings/2023-07-15.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+- See [`#347`](https://github.com/GlasgowEmbedded/glasgow/issues/347)
+
+
+## Outstanding Topics
+
+### 1. Teach Glasgow to cache applets
+
+Assignee: nobody
+
+- Depends on `1`, @attie to investigate once complete
```
Comment (mwkmwkmwk) [inline docs/meetings/2023-07-15.md:68]: but this *is* point 1?

Comment (attie) [reply docs/meetings/2023-07-15.md:68]: Ahem, yes... this might start to get messy...

Perhaps next week I'll reference by title, not number.

**Review by whitequark — APPROVED** [review]

**Pull request merged**